src/component/Form.js

```diff
@@ -31,12 +32,16 @@ class Form extends React.Component {
         window.alert(response.result.error.message)
       } else {
         const sortedVideos = response.result.items.sort((a, b) => (parseInt(a.statistics.viewCount) < parseInt(b.statistics.viewCount)) ? 1 : -1)
+        this.setState({ showError: false })
```

Comment:
why? what logic?

Reply:
if the logic shown first it has to be turn off if back to normal